### ISSUE TITLE
How to show table in Uncontrolled Carousel ?

### ISSUE THREAD

**Issue by bilalbutt044** [reactstrap/reactstrap#1553]
### Issue description
I am trying to display table in Uncontrolled Carousel instead of images but it is not showing anything. I couldn't understand what key values i would have  to write in items array to show table . below is my code. 

```
[import React from 'react';
import { UncontrolledCarousel, Table } from 'reactstrap';

const table = (
  <Table>
  <thead>
    <tr>
      <th>#</th>
      <th>First Name</th>
      <th>Last Name</th>
      <th>Username</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">1</th>
      <td>Mark</td>
      <td>Otto</td>
      <td>@mdo</td>
    </tr>
    <tr>
      <th scope="row">2</th>
      <td>Jacob</td>
      <td>Thornton</td>
      <td>@fat</td>
    </tr>
    <tr>
      <th scope="row">3</th>
      <td>Larry</td>
      <td>the Bird</td>
      <td>@twitter</td>
    </tr>
  </tbody>
</Table>
)
const items = [
  {
    Table: table,
  }
];

const Example = () => <UncontrolledCarousel items={items} />;

export default Example;](url)

```

it doesn't being shown any error but it also not showing table.

**Comment by bpas247** [comment]
```js
export default Example;](url)
```
That doesn't seem like valid JS code... Either way, that's not the correct way to use `UncontrolledCarousel`.
https://github.com/reactstrap/reactstrap/blob/b4edeb8c417aaa7e8d6370e3fd242612c2eae73c/src/UncontrolledCarousel.js#L62-L73

This is what `UncontrolledCarousel` does with the `items` prop being passed in, so passing in `items` defined as
```js
const items = [
  {
    Table: table,
  }
];
```
Will simply not work :) I do agree that the docs are a little misleading on what `items` should actually be, so maybe we should make the prop type for `items` to be more explicit than just an array?

**Comment by bilalbutt044** [comment]
@bpas247 you mean i have to explicitly define props type of items array to table ?

**Comment by bilalbutt044** [comment]
@bpas247 Can you please provide sample code ?

**Comment by bpas247** [comment]
> you mean i have to explicitly define props type of items array to table ?
 
Nope, what I’m saying is that the way you want the UncontrolledCarousel API to work with Tables is probably not going to work like that, since it’s expecting a specific shape for items than what you have currently. Hence why I suggested that we should open a new PR to change the prop “items” be more explicit with its shape.

 > Can you please provide sample code?

Sample code on what exactly? I already stated it’s probably not possible to render the Table components like you’re expecting it to render inside of UncontrolledCarousel. Do you even want to use Table here?

**Comment by bilalbutt044** [comment]
I just want to show table in slide instead of images.

**Comment by bpas247** [comment]
> I just want to show table in slide instead of images.

Afaik that's not currently supported by the `Carousel` upstream component, so I doubt we would add support for that in either `Carousel` or `UncontrolledCarousel` :)
